code/game/gamemodes/abduction/abduction.dm

```diff
@@ -24,7 +25,8 @@
 	world << "<b>Crew</b> - don't get abducted and stop the abductors."
 
 /datum/game_mode/abduction/pre_setup()
-	abductor_teams = max(1, min(max_teams,round(num_players()/config.abductor_scaling_coeff)))
+	// abductor_teams = max(1, min(max_teams,round(num_players()/config.abductor_scaling_coeff))) Old scaling
+	abductor_teams = max(1, min(max_teams,round(max_teams*num_players()/max_players))) // Scaling based off of max players
```

Comment:
On second thought the old scaling isn't too bad, 8 antags in a 45 player round isn't the most sound of ideas.

It's also because the thought of touching config files makes me want to eat rocks.